ragna/deploy/_ui/central_view.py

```diff
@@ -123,12 +124,37 @@ def __init__(
             ),
         )
 
+        # we make this available on the instance so that we can update the value later
+        self.clipboard_button = CopyToClipboardButton(
+            value=self.content_pane.object,
+            title="Copy",
+            stylesheets=[
+                ui.CHAT_INTERFACE_CUSTOM_BUTTON,
+            ],
+        )
+
+        # we make this available on the instance so that we can toggle the visibility
+        self.toolbar = pn.Row(
```

Comment:
I'm not against the toolbar name, but should we make it more obvious that this only applies to assistant messages?

Reply:
Renamed as `assistant_toolbar`.